package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@temporalio/ui",
-  "version": "2.1.13",
+  "version": "2.1.14",
```

Comment:
I just bumped to 2.1.14, let's make this 2.1.15

Reply:
Otherwise looks good!

Reply:
Is [this commit](https://github.com/temporalio/ui/pull/749/commits/78b170ca950e59f68c4ad341609c4ec2c5b0f477) autogenerated? I definitely don't remember adding it 😅

Reply:
Doesn't get autogenerated but main might have had that version already but it wasn't published yet

Reply:
The general rule of thumb we try to follow is if a PR has updates for the npm package, bump the version number, get a review, and if approved merge and then publish

Reply:
Got it, thanks! I think will actually leave this for now and bump it in my next PR that will be up shortly.